packages/core/src/components/tabs/test/folder-tabs/folder-tabs.e2e.ts

```diff
@@ -33,7 +33,9 @@ test.describe.parallel('tds-folder-tabs', () => {
     await expect(folderTabs.page()).toHaveScreenshot({ maxDiffPixels: 0 });
   });
 
-  test('selected tab index should be 0', async () => {
+  test('selected tab index should be 0', async ({ page }) => {
+    await page.waitForChanges();
```

Comment:
Nice one! I did not know about this one. I think we need to add it to more tests.